webpack.config.cjs

```diff
@@ -304,6 +312,17 @@ const templateParameters = (compilation, assets, assetTags, options) => ({
 
 const editorAndDashboard = {
   ...sharedConfig,
+  devServer: !isProduction
+    ? {
+        devMiddleware: {
+          writeToDisk: true,
+        },
+        hot: true,
+        allowedHosts: 'all',
+        host: 'localhost',
+        port: 8080,
```

Comment:
@samwhale I see https://github.com/GoogleForCreators/web-stories-wp/pull/10779#issuecomment-1074335024, but why are we specifically using port `8080` ?

Reply:
I was seeing an error about the web socket connection failing. `auto` wasn't working for me locally for some reason, even though it should have.

> I did use `port: 'auto'` though without any issues.
> 
> FWIW I am also using localwp.com for local development and not the Docker container from the repo—maybe that makes a difference.

I'm using the docker container - so maybe it had to do with that?